data/calibration.htm

```diff
@@ -187,7 +187,7 @@ <h5 class="card-title">Calibrate Tilt</h5>
                                         class="form-control" placeholder="0" readonly>
                                 </div>
                                 <div class="col-sm-2">
-                                    <button type="submit" class="btn btn-primary" name="quadratic" value="quadratic">Apply</button>
+                                    <button type="submit" id="applySettings" class="btn btn-primary" name="quadratic" value="quadratic">Apply</button>
```

Comment:
You've got multiple elements in this with the same ID, which I think is a no-no. See my comment below.

Reply:
Resolved with 0ed7a21.